VideoLocker/res/values/text_styles.xml

```diff
@@ -157,7 +157,7 @@
         <item name="android:layout_height">wrap_content</item>
         <item name="android:gravity">center</item>
         <item name="android:layout_centerInParent">true</item>
-        <item name="android:textColor">@color/new_user_txt</item>

```

Comment:
I think that `new_user_txt` and `new_user_txt_color` are now no longer being used now, so they can be removed.